### PR TITLE
fixing fetching invitations for specific organization

### PR DESCRIPTION
**Pull request by Ndevu12**
# PR Description

This PR includes fix on filtering users based on a specific organisation as well as specific admin

# Description of tasks that were expected to be completed
1. Filtering based on the organisation name

# How has this been tested?

1. Clone the repository using `git clone`
Navigate to the branch using `git checkout fx-invitation-filter`
Run `npm run dev` in the terminal

2. Using [this](https://metron-devpulse-git-fx-invitation-filter-metron.vercel.app/) link
Use admin credentials:devpulse@proton.me password:Test@12345
# Number of Commits
1 commit

# Screenshots (If appropriate)
a1
a2

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] My test coverage meet the set test coverage threshold
- [x] There are no vulnerabilities
- [x] There are no conflicts with the base branch

### CODE REVIEW

**Review by niyobertin — COMMENTED** [review]
Resolve conflicts

**Comment by Ndevu12** [comment]
> Resolve conflicts

Resolved

**Review by YvetteNyibuka — APPROVED** [review]

**Review by Oliviier-dev — APPROVED** [review]

**Review by UwicyezaG — APPROVED** [review]

**Review by niyobertin — APPROVED** [review]

**Review by ncutixavier — COMMENTED** [review]
This PR has many unnecessary changes caused by formatter

File: src/pages/invitation.tsx
Comment (ncutixavier) [inline src/pages/invitation.tsx:1]: Most changes in this file are formatting. Is it part of the task? if not, Please remove all unnecessary changes

**Review by ncutixavier — APPROVED** [review]
